build.cmd

```diff
@@ -23,7 +23,7 @@ set "__LogsDir=%__RootBinDir%\Logs"
 set __MSBCleanBuildArgs=
 set __SkipTestBuild=
 set __ToolchainMilestone=testing
-set __DotNetCliPath=
+set "__DotNetCliPath=%__ProjectDir%\Tools\dotnetcli"
 
```

Comment:
This would prevent us from rapidly iterating on ILC related changes to `dotnet compile-native`, we might have to wait for  MSBuild to update CLI. Ultimately, `compile-native` is moving to our repo, but until then, if we want PRs like multimodule support to `dotnet compile-native` to be consumed here, then we should continue to download latest available CLI that contains our ILC related changes.